src/core/scene/a-scene.js

```diff
@@ -317,10 +395,33 @@ var AScene = module.exports = registerElement('a-scene', {
 
         if (this.isPlaying) {
           this.tick(time, timeDelta);
+          this.tock(time, timeDelta);
+        }
+
+        if (this.behaviors.postRenderTick.length > 0 || this.behaviors.postRenderTock.length > 0) {
+          // Post processing setup has been modified, emit a notification.
+          if (this.postNeedsNotify) {
+            var bs = this.behaviors;
+            this.postNeedsNotify = false;
+            this.emit('post-modified', { tick: bs.postRenderTick, tock: bs.postRenderTock });
+          }
+
+          this.renderer.render(this.object3D, camera, this.renderTarget);
+
+          this.time = time;
```

Comment:
On a related note @dmarcos mentioned that he'd wanted a means to check if post processing takes too long. Should we take another timestamp with performance.now() and I don't know spew some debug with that here  or should that be done inside a component in postRenderTock?